client/src/components/UserAvatar/index.js

```diff
@@ -31,7 +39,9 @@ class UserAvatar extends PureComponent {
     }
 
     handleGetUrlSuccess = (url) => {
-      this.setState({ src: url, isLoading: false });
+      if (this._isMounted) {
```

Comment:
I know that this is a bit ugly and dirty solution, but I did not find a better way to do it for the moment.